src/available-command.ts

```diff
@@ -114,7 +114,6 @@ export function generate(engine: Engine): AvailableCommand[] {
             break;
           }
 
-          case Command.Build:
```

Comment:
> During your turn, you can take 
free actions immediately before or 
after your action. Free actions are 
explained on page 16.

I think the action meant here is the main action (i.e. using the tempstep power action), and building a mine is included in it, so you can't use a freeaction in the middle?

If you want to be able to use a free action betwen the special/power action & the build mine action, there's another change to make, because as it is, if the player chooses to use a free action instead of building a mine, he loses the ability to build the mine I think.

Reply:
Actually I've double checked the code and it's allowing also different actions. I've opened #67

Reply:
Since this seems to fix a bug, and there are no other problems, I'll merge this for now. #66 and #67 can be handled by a next PR.